### ISSUE TITLE
\texttt{>} causes an error

### ISSUE THREAD

**Issue by jsmaniac** [Khan/KaTeX#662]
In the live preview at https://khan.github.io/KaTeX/, entering any of the following causes an error:

```
\texttt{>}
\textit{>}
\textrm{>}
\textbf{>}
\text{>}

\texttt{->} and so on.
```

The error is:
```
KaTeX parse error: Expected '}', got '>' at position 9: \texttt{>̲}
```

**Comment by xymostech** [comment]
What do you expect to happen in these cases? Putting `\text{>}` into LaTeX gives me an upside down question mark.

![image](https://cloud.githubusercontent.com/assets/217900/23285109/2142ffbc-f9e4-11e6-8f03-7bd8535caa27.png)

**Comment by jsmaniac** [comment]
@xymostech Good point, I had forgotten TeX's bogus behaviour for these commands. The `\texttt{>}` does print fine with LaTeX though (and that's what I was using). Incidentally, the “correct” form `\textrm{\textgreater}` also fails with KaTeX.

**Comment by jsmaniac** [comment]
Also, adding `\usepackage[T1]{fontenc}` to the preamble of a LaTeX document makes the other commands produce the expected `>` symbol, so i'd argue that should be KaTeX's behaviour as it is very common to use `T1`, and the behaviour seems more reasonable than the upside-down question mark.

**Comment by xymostech** [comment]
Oh, I guess I didn't actually check `\texttt{>}`, you're right, that does work correctly in LaTeX! Thanks for reporting this! :)

**Comment by edemaine** [comment]
Fix forthcoming in #684.
